plane/source/docs/automatic-landing.rst

```diff
@@ -300,279 +300,234 @@ minimise barometric error problems with auto-land
 -  fly shorter flights, allowing for less time for airpressure changes.
    Check your logs and see if the landing is happening at zero altitude
    consistently
-
+
 With planes that belly land it can also work well to setup the landing
 with a shallow pitch (in ``LAND_PITCH_CD``) and set a slightly higher
 altitude to flare at. That will only work if your stall speed is low
 enough that gliding for a while will work reliably.
-
+
 Using DO_LAND_START
 =====================
-
+
 Sometimes it is useful to trigger an automatic landing as part of an RTL
 (return to launch). To do this you need to do two things:
-
+
 -  add a :ref:`DO_LAND_START <mav_cmd_do_land_start>`
    mission item to your mission, just before the start of your landing
    sequence
 -  set the :ref:`RTL_AUTOLAND <RTL_AUTOLAND>`
    parameter to 1 or 2
-
+
 The way it works is that when the plane enters an RTL it checks to see
 if the parameter RTL_AUTOLAND is set to 1 or 2. If it is then the
 current mission is searched for a mission item of type DO_LAND_START.
 If one is found then the plane will automatically enter AUTO mode and
 land, starting at the part of the mission just after the
 ``DO_LAND_START`` marker.
-
+
 The exact behaviour depends on the ``RTL_AUTOLAND`` value:
-
+
 -  If ``RTL_AUTOLAND=1`` then the plane will first RTL as normal, then
    when it starts circling the return point (home or a rally point) it
    will then switch to the AUTO mission after the ``DO_LAND_START`` and
    land
 -  If ``RTL_AUTOLAND=2`` then the plane will bypass the RTL completely
    and go straight to the landing sequence.
-
+
 You can optionally include more than one ``DO_LAND_START`` mission item
 in your mission. If that is done then the latitude/longitude of the
 ``DO_LAND_START`` mission items is used to choose which landing sequence
 to use. The ``DO_LAND_START`` closest to the current location is used.
 This can be useful if you have multiple landing sequences for different
 wind conditions or different areas.
-
-How to manually abort an auto-landing
+
+How to abort an auto-landing
 =====================================
-
-A landing-abort mechanism is provided to allow you to abort a landing
-sequence in a safe, controlled, and expected way. Custom abort behaviour
-can be pre-programmed as part of the mission or you can use the default
-abort recovery mechanism. To enable, set param LAND_ABORT_THR=1.
-
-.. note::
-
+A landing-abort mechanism is provided to allow you to abort a landing sequence in a safe, controlled, and expected way. Custom abort behaviour can be pre-programmed as part of the mission or you can use the default abort mechanism. To enable this feature set param LAND_ABORT_THR=1.
+ 
+There are three steps to this feature:
+Step 1) Trigger an abort

```

Comment:
These should still be list items. IMO should look as below. The spacing below is required.
```
There are three steps to this feature:

#. Trigger an abort
#. The behavior during the abort
#. The mission state after the abort completes.
```